.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
-max-line-length = 80
+ignore = E203, E266, E501, W503, F403, F401
+max-complexity = 30
+max-line-length = 79
```

Comment:
79 seems very ungenerous - I like 120 these days. IS there any particular motivation for this stringency?

Reply:
Happy to bump this up to 120 in 500e894